### PR TITLE
Jump to docs

### PR DESCRIPTION
**Pull request by garrensmith**
Replace the jump to docs backbone view with a React one

### CODE REVIEW

File: app/addons/documents/jumptodoc.react.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    return (
+      <div id="jump-to-doc" class="input-append">
+        <ReactSelect
+          name="jump-to-doc"
+          placeholder="Document ID"
+          className="input-large jump-to-doc"
+          options={options}
+          clearable={false}
+          onChange={({value: docId}) => {
+            var url = FauxtonAPI.urls('document', 'app', app.utils.safeURLName(database.id), app.utils.safeURLName(docId) );
```
Comment (robertkowalski) [inline app/addons/documents/jumptodoc.react.jsx]: const

File: app/addons/documents/assets/less/jumptodoc.less [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#jump-to-doc-wrapper {
+}
+
+.Select-menu-outer {
+}
```
Comment (robertkowalski) [inline app/addons/documents/assets/less/jumptodoc.less]: remove those and add license header

File: app/addons/documents/jumptodoc.react.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+define([
+  '../../app',
+  '../../core/api',
+  'react',
+  'react-dom',
+  'react-select',
+  'lodash'
+], (app, FauxtonAPI, React, ReactDOM, ReactSelect, {debounce}) => {
+
+  const navToDoc = ({value: docId}) => {
```
Comment (robertkowalski) [inline app/addons/documents/jumptodoc.react.jsx]: this is unused

File: app/addons/documents/jumptodoc.react.jsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      <div id="jump-to-doc" class="input-append">
+        <ReactSelect
+          name="jump-to-doc"
+          placeholder="Document ID"
+          className="input-large jump-to-doc"
+          options={options}
+          clearable={false}
+          onChange={({value: docId}) => {
+            var url = FauxtonAPI.urls('document', 'app', app.utils.safeURLName(database.id), app.utils.safeURLName(docId) );
+            FauxtonAPI.navigate(url, {trigger: true});
```
Comment (robertkowalski) [inline app/addons/documents/jumptodoc.react.jsx]: maybe use `navToDoc `

File: app/addons/documents/jumptodoc.react.jsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  };
+
+  const JumpToDoc = ({database, allDocs}) => {
+    const options = allDocs.map(doc => {
+      return {
+        value: doc.get('_id'),
+        label: doc.get('_id')
+      };
+    });
+    return (
```
Comment (robertkowalski) [inline app/addons/documents/jumptodoc.react.jsx]: maybe inject the already created optionlist into the rendering component? could that make it reusable for other dropdown situations, like jumpt-to-database?

Comment (garrensmith) [inline app/addons/documents/jumptodoc.react.jsx]: I cannot find the option list. At this stage I'm not sure its worth making reusable until we have a reason to reuse it.

File: app/addons/documents/jumptodoc.react.jsx (hunk 4 above)
Comment (garrensmith) [inline app/addons/documents/jumptodoc.react.jsx]: I originally wanted to use `navToDoc` but I need some of the props passed in which makes it really tricky. I've decided to remove the `navToDoc` function.

**Comment by robertkowalski** [comment]
+2 great!

**Comment by garrensmith** [comment]
Merged.

**Pull request closed without merging**